### DISCUSSION TITLE
Modal onRuxmodalclosed is returning "e.detail : null"

### DISCUSSION THREAD

**Discussion by jldayhoff** [RocketCommunicationsInc/astro#420] [Q&A] [answered]
Upon using the example here: https://codesandbox.io/s/condescending-breeze-7c2dy?file=/src/App.js .  Confirm or Cancel produces a "user canceled" because e.detail is always "null". 

```
CustomEvent {isTrusted: false, detail: null, type: 'ruxmodalclosed', target: rux-modal.hydrated, currentTarget: rux-modal.hydrated, …}
isTrusted: false
bubbles: true
cancelBubble: false
cancelable: true
composed: true
currentTarget: null
defaultPrevented: false
detail: null
eventPhase
```

**Comment by micahjones13** [comment, marked as answer]
Thanks for bringing this up! A fix for this was just merged and will be included in our next release on June 9th.
